submission/notifications/src/main/kotlin/ebi/ac/uk/notifications/api/RtClient.kt

```diff
@@ -51,21 +50,26 @@ class RtClient(
         return ticketIdPattern.match(body.second())?.secondGroup() ?: throw InvalidTicketIdException()
     }
 
-    private fun ticketComment(ticketId: String, comment: String) =
-        StringBuilder("id: $ticketId\n")
-            .append("Action: correspond\n")
-            .append("Status: resolved\n")
-            .append("Text: ${trimContent(comment)}")
-            .toString()
+    private fun ticketComment(ticketId: String, comment: String): String {
+        return buildString {
+            appendLine("id: $ticketId")
+            appendLine("Action: correspond")
+            appendLine("Status: resolved")
+            append("Text: ${trimContent(comment)}")
+        }
+    }
 
-    private fun ticketContent(accNo: String, subject: String, owner: String, content: String) =
-        StringBuilder("Queue: ${rtConfig.queue}\n")
-            .append("Subject: $subject\n")
-            .append("Status: resolved\n")
-            .append("Requestor: $owner\n")
-            .append("CF-Accession: $accNo\n")
-            .append("Text: ${trimContent(content)}")
-            .toString()
+    private fun ticketContent(accNo: String, subject: String, owner: String, ccUser: String?, content: String): String {
+        return buildString {
+            appendLine("Queue: ${rtConfig.queue}")
+            appendLine("Subject: $subject")
+            appendLine("Status: resolved")
+            appendLine("Requestor: $owner")
+            ccUser?.let { appendLine("AdminCc: $it") }
```

Comment:
I prefered to allow to flexbility to disable/enable this feature for example in some environments or during some process when we do not want to recieve a copy.  Clearly it add like 0.1% extra complecity but I do not think is 100% we are doing BCC in envery environment. 

Still not 100% sure so let me know if you are 100% in with being mandatory

Reply:
I was mostly concerned about us forgetting about it and because it isn't required, the app would start normally but you're right, we only need this for PROD and maybe BETA notifications, we don't care about other environments and it should be enough by having it in the config file